canonical/ubuntu.com
# WD-13861 - create a function for form execution

## Done

Created a function for form execution

REMOVE FORM FROM BASE INDEX AND AWS AFTER TESTING

## QA

- Check the form on the demo: [static](https://ubuntu-com-14215.demos.haus/form-template) and [dynamic](https://ubuntu-com-14215.demos.haus/aws#get-in-touch)
- Make sure that the form opens correctly and has all needed fields (you can check expected field sets in form-data.json)
- Check that the form submits necessary data
- Check that the form has correct required fields and does not submit if a required field is empty
- Ignore GH checks for now, the testing template will be removed before merge and the checks will be successful

## Issue / Card
[WD-13861](https://warthogs.atlassian.net/browse/WD-13861)

[WD-13861]: https://warthogs.atlassian.net/browse/WD-13861?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
